### PR TITLE
Dockerize the app

### PR DESCRIPTION
**Pull request by azlyth**

### CODE REVIEW

**Comment by azlyth** [comment]
@BlaneCordes I imagine this breaks the Heroku integration, so I won't merge until we figure out a deployment strategy for the Dockerized version of this app.

**Comment by BlaneCordes** [comment]
@azlyth sweet! thanks, will check this out

File: client/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "client",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:3001",
+  "proxy": "http://api:3001",
```
Comment (BlaneCordes) [inline client/package.json]: does this work?

Comment (azlyth) [reply client/package.json]: Sure does. Each Docker Compose service is addressable by its name.

**Pull request merged**